### ISSUE TITLE
Plots section collapse review

### ISSUE THREAD

**Issue by maxagin** [iterative/vscode-dvc#2263]
1. Do not collapse the section when clicking on the empty space on the right side of the title, but collapse only when clicking on the chevron-shaped mark beside the section title.
2. When a chevron-shaped mark hovered, show pointer (cursor)
3. Hover over the information mark (i) on the right side of the section title open the tooltip -> select the text of the tooltip -> release the left mouse button, the entire section is collapsed 
4. Select the title to copy -> release the left mouse button, the entire section is collapsed 
5. Select the title to copy -> right click is not working, no context menu with the text copy option is opened

**Comment by sroy3** [comment]
> 1. Do not collapse the section when clicking on the empty space on the right side of the title, but collapse only when clicking on the chevron-shaped mark beside the section title.

I don't mind changing that, ~~but I just want to point out that we are currently using the built-in mechanism of the [`<details>` ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) HTML element for the collapse behaviour.~~ I think it would be possible to keep the current behaviour (less code, and more performant) and simply reducing the width of the `<summary>` element (the section title).

Other points should be possible to fix easily as I've already done something similar elsewhere.

**Comment by maxagin** [comment]
Great! Another thing to mention: does the section title needs to be part of the trigger or only the chevron-shaped mark? If one day (and it is probably coming soon) we will be able to modify or copy the title, better if it won't be part of the section collapse trigger.

**Comment by maxagin** [comment]
Sound good. Thank you @sroy3 !

**Comment by maxagin** [comment]
- I have noticed that when the section is collapsed and you click on the chevron it takes somehow more time to uncollapse the section than collapse.

- Hovering over the info icon -> Copy text -> Right click won't do anything. I would expect to have the context menu with the copy option.

> 5. Select the title to copy -> right click is not working, no context menu with the text copy option is opened

- It still not working. Is there a reason?

- I would like only to copy the title -> double click -> section is collapsed. This is confusing as I did not want to collapse, the section, but only copy the text. Any suggestions here?
